pep/tests/integration/pods/components/document/actions/download/component-test.ts

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | document/actions/download', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`{{document/actions/download}}`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      {{#document/actions/download}}
+        template block text
+      {{/document/actions/download}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
```

Comment:
might have some failing tests here (and below) since im guessing most of these dont have block forms